salt/beacons/twilio_txt_msg.py

```diff
@@ -6,10 +6,15 @@
 # Import Python libs
 from __future__ import absolute_import
 import logging
+from salt.ext.six.moves import map
 
 # Import 3rd Party libs
 try:
-    from twilio.rest import TwilioRestClient
+    import twilio
+    if twilio.__version__ > 5:
```

Comment:
Is `__version__` a string?
If its a string, you can't compare strings to ints in py 3.
Is it a Tuple of ints?
Then maybe `> (5,)`
It's an int? Nevermind this review comment.